packages/forms-web-app/src/middleware/check-decision-date-deadline.js

```diff
@@ -10,16 +10,7 @@ const {
 		APPELLANT_SUBMISSION: { CANNOT_APPEAL: cannotAppealHAS }
 	}
 } = require('../lib/views');
-const validationExclusionPages = [
```

Comment:
felt safe to just rely on /before-you-start to simplify this
user will be redirected to a cannot appeal page if they enter a date beyond the deadline in decision-date
So think the only way this would impact someone is if they changed the appeal type after setting date, and typed the urls rather than followed the journey, at which point they would get told after completing before you start